opentech/apply/projects/views.py

```diff
@@ -162,9 +184,11 @@ def form_valid(self, form):
             source=project,
         )
 
+        project.send_to_compliance(self.request)
+
         project.is_locked = False
         project.status = CONTRACTING
-        project.save(update_fields=['is_locked', 'status'])
+        project.save(update_fields=['is_locked', 'sent_to_compliance_at', 'status'])
```

Comment:
`sent_to_compliance_at` is handled in the `send_to_compliance` method.

Reply:
Fixed and rebased.